sdk/core/core-lro/test/lro.spec.ts

```diff
@@ -1077,6 +1077,47 @@ matrix(
               assert.equal(result.id, "100");
             });
 
+            it("should handle patchAsyncNoLocationHeader", async () => {
+              const resourceLocationPath = `/patchasyncnolocationheader/succeeded`;
+              const initialResourcePath = resourceLocationPath;
+              const pollingPath = `/patchasyncnolocationheader/operationresults/123`;
+              const result = await runLro({
+                routes: [
+                  {
+                    method: "PATCH",
+                    status: 202,
+                    path: initialResourcePath,
+                    headers: {
+                      [headerName]: pollingPath,
+                    },
```

Comment:
There should be a Location header too according to the [guidelines](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#updating-using-patch).

Reply:
I think the fix is more focusing on if the service side would not exactly follow the guideline we could have a fallback logic to get the resource location based on the fact that this is a PATCH method which would work on existing resource for (partial) update.